nyaa/templates/search_results.html

```diff
@@ -52,7 +52,7 @@
 				{% set icon_dir = config.SITE_FLAVOR %}
 				<td style="padding:0 4px;">
 				{% if use_elastic %}
-				<a href="/?c={{ cat_id }}" title="{{ torrent.main_category_id }} - {{ torrent.sub_category_id }}">
+				<a href="/?c={{ cat_id }}" title="{{ torrent.main_category.name }} - {{ torrent.sub_category.name }}">
```

Comment:
This doesn't actually work. The ES rows don't store the name, just the ID.
I know it's not easy to test the ES functionality as it requires a lot of setup, but if you can't test it (or aren't perfectly sure you know how it works), please leave it to those who have it set up.